locales/en/app.ftl

```diff
@@ -24,8 +24,10 @@
 -brand-github = GitHub
 -brand-mozilla-vpn = Mozilla VPN
 -brand-relay = Firefox Relay
+-brand-relay-premium = Relay Premium
 -brand-mozilla-monitor = Mozilla Monitor
 -brand-monitor-plus = Monitor Plus
+-product-name-privacy-products-bundle = Privacy Protection Plan
```

Comment:
Both these should go in the locales-pending folder, no point in translating them now.

This one in particular needs discussions and clear guidance from content/product.